frontend/src/tests/components/eMIB/EditActionDialog.test.js

```diff
@@ -263,3 +268,125 @@ it("clicking on the button only adds the email once", () => {
   expect(handleClose).toHaveBeenCalledTimes(1);
   expect(addEmail).toHaveBeenCalledTimes(1);
 });
+
+describe("edit action dialog helper file", () => {
+  // ============================== VARIABLES ==============================
+  // empty variables
+  const emptyEmailType = "";
+  const emptyEmailTo = "";
+  const emptyEmailCc = "";
+  const emptyEmailResponse = "";
+  const emptyReasonsForActionContent = "";
+  const emptyTaskContent = "";
+
+  // initial variables (for edit only)
+  const initialEmailType = EMAIL_TYPE.replyAll;
+  const initialEmailTo = "to";
+  const initialEmailCc = "cc";
+  const initialEmailResponse = "response";
+  const initialReasonsForActionContent = "reasons for action";
+  const initialTaskContent = "tasks";
+
+  // current variables
+  let emailType = EMAIL_TYPE.replyAll;
```

Comment:
you should make use of beforeEach and afterEach to reset your variables that get modified. this will guarantee that the order you run your tests in does not matter.
https://github.com/code-for-canada/project-thundercat/blob/master/frontend/src/tests/modules/EmibInboxRedux.test.js#L17

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/229/commits/7cda64a26be2e2bc767893ebe9203a05266bfc85